packages/http/src/utils/runtimeExpression.ts

```diff
@@ -111,16 +108,14 @@ export function resolveRuntimeExpression(
 
   function readBody(body: unknown) {
     return pipe(
-      doOption
-        .bind('body', O.fromNullable(body))
-        .bind(
-          'path',
-          pipe(
-            lookup(2, parts),
-            O.chain(part => O.tryCatch(() => pointerToPath('#' + part)))
-          )
+      O.fromNullable(body),
+      O.bindTo('body'),
+      O.bind('path', _ =>
```

Comment:
```suggestion
      O.bind('path', () =>
```

Not sure here, but I feel that this argument is commonly skipped so there is no reason to emphasize that it exists.